integration-tests/runJest.js

```diff
@@ -45,12 +45,11 @@ function runJest(
     );
   }
 
-  const env = options.nodePath
-    ? Object.assign({}, process.env, {
-        FORCE_COLOR: 0,
-        NODE_PATH: options.nodePath,
-      })
-    : process.env;
+  const env = Object.assign({}, process.env, {
```

Comment:
Why these changes? Maybe reverting fixes the snapshots?

Reply:
No, when I don't do this change I get lots of color codes in the test snapshots :-/

Reply:
You're right, by deleting it and running jest from root it doesn't have that problem with colors. Huh.